From d19464ff25cd59db657061b901f97d2cf9fff800 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Tue, 4 Jun 2013 10:50:27 +0900
Subject: [PATCH 529/675] LTSI: ARM: shmobile: bockw: enable
 CONFIG_EXPERIMENTAL

It is required for MTD_M25P80 and ARM_APPENDED_DTB

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
---
 arch/arm/configs/bockw_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/bockw_defconfig b/arch/arm/configs/bockw_defconfig
index 845f5cd..1d156dc 100644
--- a/arch/arm/configs/bockw_defconfig
+++ b/arch/arm/configs/bockw_defconfig
@@ -1,4 +1,5 @@
 # CONFIG_ARM_PATCH_PHYS_VIRT is not set
+CONFIG_EXPERIMENTAL=y
 CONFIG_KERNEL_LZMA=y
 CONFIG_NO_HZ=y
 CONFIG_IKCONFIG=y
-- 
1.8.2

